variables.tf

```diff
@@ -15,6 +15,11 @@ variable "apiary_tags" {
   type        = map(any)
 }
 
+variable "apiary_extra_tags_s3" {
+  description = "Additional tags for Apiary S3 buckets."
+  type        = map(any)
+}
```

Comment:
perhaps you should add a default value so this variable is optional
default = null

Reply:
does the `merge` method handle nulls gracefully? if not default should be empty map

Reply:
Merge function in terraform does not handle null values gracefully, so I have defined an empty map as default. Thanks!